webapp/.env.development

```diff
@@ -1 +1 @@
-REACT_APP_API_URL="https://localhost:5001/api"
\ No newline at end of file
+REACT_APP_API_URL=http://localhost:3001
```

Comment:
Do not change this.  It should point to the DotNet API by default.  Instead, add a comment that shows how to point the React front end to the API mocks.